src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/SafeHandles.cs

```diff
@@ -12,22 +13,19 @@ namespace Internal.Cryptography.Pal.Native
     /// </summary>
     internal abstract class SafePointerHandle<T> : SafeHandle where T : SafeHandle, new()
     {
-        public SafePointerHandle()
+        protected SafePointerHandle()
             : base(IntPtr.Zero, true)
         {
         }
 
-        public sealed override bool IsInvalid
+        internal static T CreateInvalid()
```

Comment:
I disagree. I put it here because it's a factory and I believe should be grouped with the ctors. If the resulting diff is not ideal, that's lead about the change and more about the diff tool.